plugins/portals-list.js

```diff
@@ -168,6 +209,9 @@ window.plugin.portalslist.getPortals = function() {
       default:
         window.plugin.portalslist.neuP++;
     }
+    if (portal.options.data.history.visited) window.plugin.portalslist.visitedP++;
+    if (portal.options.data.history.captured) window.plugin.portalslist.capturedP++;
+    if (portal.options.data.history.scoutControlled) window.plugin.portalslist.scoutControlledP++;
```

Comment:
```suggestion
['visited', 'captured', 'scoutControlled'].forEach(function (prop) {
  if (portal.options.data.history[prop]) {
    window.plugin.portalslist[prop+'P']++; 
  }
});
```

Reply:
simplicity vs. smartness.

Reply:
I disagree on the `+'P'`
smart one may rewrite all counters in a different way
current is good enough and leaves the PR consistent

Reply:
> smart one may rewrite all counters in a different way

Sure, let's leave that for another PR.